### DISCUSSION TITLE
服务模式打不开是什么情况

### DISCUSSION THREAD

**Discussion by cancundeyingzi** [zzzgydi/clash-verge#861] [Q&A]
已用管理员权限
![image](https://github.com/zzzgydi/clash-verge/assets/73635883/38bb2116-28a0-4496-ae99-9e8620e1a6cc)

![image](https://github.com/zzzgydi/clash-verge/assets/73635883/cf984197-bc16-48c3-9256-662efefc42c4)

**Comment by zzzgydi** [comment]
在任务管理器里把旧的卸了，再试试

**Reply by cancundeyingzi** [reply]
测试无效,重启电脑依旧自动关闭,开了系统代理这个聊天框都打不开(已经反复刷新网页),开了tun模式,立即好了

**Reply by cancundeyingzi** [reply]
也就是目前想用meta核心,只能关开机自起,手动管理员运行,开tun,别的系统代理什么都用不了

**Reply by zzzgydi** [reply]
管理员权限执行 `sc delete clash_verge_service`

**Reply by zzzgydi** [reply]
https://github.com/zzzgydi/clash-verge/issues/355
